.github/workflows/build-wheels.yaml

```
@@ -92,3 +91,74 @@ jobs:
                 ls /tests/test_*.py | sed 's/\.py$//' | sed 's/\/tests\//tests./' | CI=1 xargs --verbose -I {} python -m unittest {}
               "
           done
+
+  build-arm:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04-arm
```
Then you don't need qemu emulation.

https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories says:
> For public repositories, jobs using the workflow labels shown in the table below will run on virtual machines with the associated specifications. The use of these runners on public repositories is free and unlimited.

---

oh cool, ill test this on a separate branch and see if I can get it to work. I noticed building and testing the wheels with QEMU emulation takes much longer, like a couple hours, so hopefully this cuts down on that.

---

this works great, added in latest commit.